examples/tutorials/curation/plot_2_train_a_model.py

```diff
@@ -0,0 +1,125 @@
+
+both_sortings = si.aggregate_units([sorting_1, sorting_2])
+
+##############################################################################
+# The models are based on `quality metrics <https://spikeinterface.readthedocs.io/en/latest/modules/qualitymetrics.html>`_
+# and `template metrics <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html#template-metrics>`_,
+# which are computed using a :code:`sorting_analyzer`. So we'll now create a sorting
+# analyzer and compute the extensions needed to get the metrics.
+
+analyzer = si.create_sorting_analyzer(sorting = both_sortings, recording=recording)
+analyzer.compute(['noise_levels','random_spikes','waveforms','templates','spike_locations','spike_amplitudes','correlograms','principal_components','quality_metrics','template_metrics'])
+
+##############################################################################
+# Let's plot the templates for the first and fifth units. The first (unit id 0) belongs to
+# :code:`sorting_1` so should look like a real unit; the sixth (unit id 5) belongs to :code:`sorting_2`
+# so should look like noise.
+
+sw.plot_unit_templates(analyzer, unit_ids=[0,5])
+
+##############################################################################
+# This is as expected: great! Find out more about plotting using widgets `here <https://spikeinterface.readthedocs.io/en/latest/modules/widgets.html>`_. The labels
+# for our units are then easy to put in a list:
+
+labels = ['good', 'good', 'good', 'good', 'good', 'bad', 'bad', 'bad', 'bad', 'bad']
+
+##############################################################################
+# Step 2: Train our model
+# -----------------------
+#
+# With the labelled data, we can train the model using the :code:`train_model` function.
+# Here, the idea is that the trainer will try several classifiers, imputation strategies and
+# scaling techniques then save the most accurate. To save time, we'll only try one classifier
+# (Random Forest), imputation strategy (median) and scaling technique (standard scaler).
+
+folder = "my_model"
+
+# We will use a list of one analyzer here, we would strongly advise using more than one to
+# improve model performance
+trainer = sc.train_model(
+    mode = "analyzers", # You can supply a labelled csv file instead of an analyzer
+    labels = [labels],
+    analyzers = [analyzer],
+    folder = folder, # Where to save the model and model_info.json file
+    metric_names = None, # Specify which metrics to use for training: by default uses those already calculted
+    imputation_strategies = ["median"], # Defaults to all
+    scaling_techniques = ["standard_scaler"], # Defaults to all
+    classifiers = None, # Default to Random Forest only. Other classifiers you can try [ "AdaBoostClassifier","GradientBoostingClassifier","LogisticRegression","MLPClassifier"]
+    overwrite = True, # Whether or not to overwrite `folder` if it already exists. Default is False.
+)
```

Comment:
this was showing an error on the build docs for this PR

Reply:
Thanks - we added PC metrics and some of them had `None` default_params, which we weren't expecting.